admin/app/jobs/RebuildIndexJob.scala

```diff
@@ -96,20 +96,10 @@ class RebuildIndexJob(contentApiClient: ContentApiClient)(implicit executionCont
       contributors <- contentApiTagsEnumerator.enumerateTagType("contributor")
     } yield {
       val contributorsByNameOrder: Map[String, Set[Tag]] = contributors.toSet
-        .groupBy(tag => {
-
-          /**
-            * The alphaIndexKey function looks at the first alphanumeric character in the string passed to it.
-            * Concatenating these three strings therefore allows us to use firstName and webTitle as fallbacks
-            * if lastName is None or "".
-            * */
-          val indexString =
-            tag.lastName.getOrElse("").trim.concat(tag.firstName.getOrElse("").trim).concat(tag.webTitle)
-          tagPages.alphaIndexKey(indexString)
-        })
```

Comment:
This has been moved to the `TagPages` object from here in `RebuildIndexJob` - it's easier to test once it's a static function (rather than having to create an instance of `RebuildIndexJob` to test it), and it consolidates the alpha-index-key logic into one place.